lib/growthbook/feature_rule.rb

```diff
@@ -54,16 +88,24 @@ def is_force?
     end
 
     def to_json(*_args)
-      res = {}
-      res['condition'] = @condition unless @condition.nil?
-      res['coverage'] = @coverage unless @coverage.nil?
-      res['force'] = @force unless @force.nil?
-      res['variations'] = @variations unless @variations.nil?
-      res['key'] = @key unless @key.nil?
-      res['weights'] = @weights unless @weights.nil?
-      res['namespace'] = @namespace unless @namespace.nil?
-      res['hashAttribute'] = @hash_attribute unless @hash_attribute.nil?
-      res
+      {
+        'condition' => @condition,
+        'coverage' => @coverage,
+        'force' => @force,
+        'variations' => @variations,
+        'key' => @key,
+        'weights' => @weights,
+        'namespace' => @namespace,
+        'hashAttribute' => @hash_attribute,
+        'range' => @range,
+        'ranges' => @ranges,
+        'meta' => @meta,
+        'filters' => @filters,
+        'seed' => @seed,
+        'name' => @name,
+        'phase' => @phase,
+        'tracks' => @tracks
+      }.compact
```

Comment:
Nice 👍  Is this tested in new or existing test cases so we can make sure it still returns the same output?